jx/bdd/tekton/ci.sh

```diff
@@ -47,4 +47,5 @@ jx step bdd --versions-repo https://github.com/jenkins-x/jenkins-x-versions.git
   --tests test-upgrade-ingress \
   --tests test-create-spring \
   --tests test-quickstart-golang-http \
```

Comment:
Actually, while we're here, might as well remove `test-quickstart-golang-http` from `tekton`, since it's covered in `boot-vault`.